lib/AST/PlatformKindUtils.cpp

```diff
@@ -291,6 +294,8 @@ swift::tripleOSTypeForPlatform(PlatformKind platform) {
     return llvm::Triple::OpenBSD;
   case PlatformKind::Windows:
     return llvm::Triple::Win32;
+  case PlatformKind::Android:
+    return llvm::Triple::Linux;
```

Comment:
I don't have any experience with LLVM's support for Android as a platform so I'll leave it up to folks like @finagolfin, @compnerd, and other experts to judge whether this kind of thing looks like it makes sense.